src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Presentation/ProjectItemDetailsPageViewModel.cs

```diff
@@ -188,13 +189,29 @@ public async Task AssignToMeAndRefresh()
 
 		private async void AssignToMeHandler(IUICommand command)
 		{
-			var workItem = await WorkItem.Task;
-			await _vstsRepository.AssignToMe(workItem.Item.WorkItem);
-
-			var updatedWorkItem = await _vstsRepository.GetWorkItem(workItem.ProjectId, workItem.Item.WorkItem.Id.Value);
+			// As these are direct calls (no TaskNotifier) try/catch the error and display it in the messageDialog
```

Comment:
Should we be using TaskNotifier here?

Reply:
Well that comment is irevelant as I think about it. TaskNotifier is handy to display errors on a page, to propagates the error but that's not what we want here...

Reply:
You can update the comment accordingly then :)